Resources/views/Collector/icon.svg

```diff
@@ -0,0 +1,7 @@
+<svg viewBox="9.954 10.02 33.268 32.669" width="33.268" height="32.669">
```

Comment:
Why such weird viewbox instead of using a `0 0 24 24` like all the existing profiler icons ?